core/cap-TBD.md

```diff
@@ -0,0 +1,193 @@
+## Preamble
+
+```
+CAP: To Be Assigned
```

Comment:
Can you assign number 0054, and add it to the `core/README.md` as well under the drafts section?

Reply:
```suggestion
CAP: 0054
```